#!/usr/bin/env python
PACKAGE = "legs_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("left_hip_initial_pos", double_t, 0, "Initial position of the hip before the impulse", 30, -180,   180)
gen.add("left_knee_initial_pos", double_t, 0, "Initial position of the knee before the impulse", 30, -180,   180)
gen.add("left_ankle_initial_pos", double_t, 0, "Initial position of the ankle before the impulse", 90, -180,   180)

gen.add("right_hip_initial_pos", double_t, 0, "Initial position of the hip before the impulse", 30, -180,   180)
gen.add("right_knee_initial_pos", double_t, 0, "Initial position of the knee before the impulse", 30, -180,   180)
gen.add("right_ankle_initial_pos", double_t, 0, "Initial position of the ankle before the impulse", 90, -180,   180)

exit(gen.generate(PACKAGE, "legs_controller", "impulse_controller"))
